src/main/java/httpServer/http/request/HTTPRequest.java

```diff
@@ -0,0 +1,56 @@
+package httpServer.http.request;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class HTTPRequest {
+
+    public String method;
+    public String resource;
+    public String httpVersionNumber;
+    public Map<String, String> headers = new HashMap<>();
```

Comment:
I wouldn't initialize the headers here.

Reply:
Would be the idea be to initialize them in a method?